index.yaml

```diff
@@ -1,23 +1,104 @@
 apiVersion: v1
 entries:
   aws-ebs-csi-driver:
+  - apiVersion: v1
+    appVersion: 0.7.1
+    created: "2020-11-26T19:49:29.1581493-05:00"
+    description: A Helm chart for AWS EBS CSI Driver
+    digest: df74acb2f2a814c56106ab51d42db8889a671cd219b3d5e7f93549056a31a915
+    home: https://github.com/kubernetes-sigs/aws-ebs-csi-driver
+    keywords:
+    - aws
+    - ebs
+    - csi
+    kubeVersion: '>=1.13.0-0'
```

Comment:
based on the readme chart we should be advertising compatibility with >=1.14.0-0, IIRC the reason is that is the version with CSI 1.1.0

Reply:
This is wrong in the Chart.yaml then as that is where the info comes from so it is stuck being wrong for chart version 0.6.1.  I can update the Chart.yaml in my other PR so it will be >= 1.14.0-0 in 0.6.2 of the chart.